controllers/rack.go

```diff
@@ -616,9 +635,18 @@ func (r *SingleClusterReconciler) scaleDownRack(
 			}
 		}
 
+		// Setup roster after migration
+		if err := r.getAndSetRoster(aerospikePolicy); err != nil {
```

Comment:
If pod is not running, getAndSetRoster will fail internally. So if IsPodRunningAndReady is false, don't set roster.